# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieService.cpp
# Commit: 01daaabadcd7
# Full Hash: 01daaabadcd7a74a9ab5114b84c5bf06c9583bff
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-10 21:44:02
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r d10ae1bb47a8 -r 01daaabadcd7 netwerk/cookie/CookieService.cpp
--- a/netwerk/cookie/CookieService.cpp	Fri Jan 10 08:24:30 2025 +0000
+++ b/netwerk/cookie/CookieService.cpp	Fri Jan 10 08:24:30 2025 +0000
@@ -1808,5 +1808,14 @@
   return NS_OK;
 }
 
+NS_IMETHODIMP
+CookieService::TestGet3PCBExceptions(nsTArray<nsCString>& aExceptions) {
+  aExceptions.Clear();
+
+  mThirdPartyCookieBlockingExceptions.GetExceptions(aExceptions);
+
+  return NS_OK;
+}
+
 }  // namespace net
 }  // namespace mozilla